db/migrate/20230829134357_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.bigint :author_id
+      t.string :users
```

Comment:
- According to the project requirements and the ERD, there is no `users` column for the `posts` table. Please remove this column to avoid errors in your code 🙏